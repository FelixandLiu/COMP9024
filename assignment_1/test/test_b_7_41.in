a
11839238
137.5596
a
62292765
151.9144
a
88057904
210.0283
a
71941543
149.3444
a
30682311
-1.0529
a
94667151
138.0545
a
26796769
72.4189
a
95629628
103.2682
a
17946725
7.6248
a
7060599
81451080
59.1276
a
174805289
59083928
238.1342
a
44314384
-749.763
46.8701
a
59976562
-730.8569
229.5512
a
92502419
120.3742
a
77021664
53.1645
a
83628056
210.0429
a
43525053
-798.046
173.9621
a
27940260
112.4704
a
43846126
74.9636
a
68717141
197.1826
a
54721825
179.3292
a
18185342
176.1922
a
52922616
-961.3114
127.65
a
51095415
212.6633
a
80390967
-784.777
76.422
a
8631158
69727071
198.4
a
19755880
93.6833
a
71561079
220.6851
a
65408206
219.408
a
11375833
112.8401
a
57594955
43.0621
a
81839215
-602.985
77.843
a
23114487
175.5132
a
67772008
132.1635
a
30920739
47.2306
a
49943910
208.66
a
17545683
101.2757
a
53870469
90.9648
a
47589286
60.8493
a
91557921
224.6835
a
37830004
126.9899
p
g
q
