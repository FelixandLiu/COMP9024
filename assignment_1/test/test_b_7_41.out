Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Not valid. Enter a valid value: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Not valid. Enter a valid value: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> Enter card ID: Enter amount: Card added.
Enter command (a,g,p,q,r, h for Help)> -----------------
Card ID: 11375833
Balance: $112.84
-----------------
-----------------
Card ID: 11839238
Balance: $137.56
-----------------
-----------------
Card ID: 17545683
Balance: $101.28
-----------------
-----------------
Card ID: 17946725
Balance: $7.62
-----------------
-----------------
Card ID: 18185342
Balance: $176.19
-----------------
-----------------
Card ID: 19755880
Balance: $93.68
-----------------
-----------------
Card ID: 23114487
Balance: $175.51
-----------------
-----------------
Card ID: 26796769
Balance: $72.42
-----------------
-----------------
Card ID: 27940260
Balance: $112.47
-----------------
-----------------
Card ID: 30682311
Balance: -$1.05
Low balance
-----------------
-----------------
Card ID: 30920739
Balance: $47.23
-----------------
-----------------
Card ID: 37830004
Balance: $126.99
-----------------
-----------------
Card ID: 43525053
Balance: $173.96
-----------------
-----------------
Card ID: 43846126
Balance: $74.96
-----------------
-----------------
Card ID: 44314384
Balance: $46.87
-----------------
-----------------
Card ID: 47589286
Balance: $60.85
-----------------
-----------------
Card ID: 49943910
Balance: $208.66
-----------------
-----------------
Card ID: 51095415
Balance: $212.66
-----------------
-----------------
Card ID: 52922616
Balance: $127.65
-----------------
-----------------
Card ID: 53870469
Balance: $90.96
-----------------
-----------------
Card ID: 54721825
Balance: $179.33
-----------------
-----------------
Card ID: 57594955
Balance: $43.06
-----------------
-----------------
Card ID: 59083928
Balance: $238.13
-----------------
-----------------
Card ID: 59976562
Balance: $229.55
-----------------
-----------------
Card ID: 62292765
Balance: $151.91
-----------------
-----------------
Card ID: 65408206
Balance: $219.41
-----------------
-----------------
Card ID: 67772008
Balance: $132.16
-----------------
-----------------
Card ID: 68717141
Balance: $197.18
-----------------
-----------------
Card ID: 69727071
Balance: $198.40
-----------------
-----------------
Card ID: 71561079
Balance: $220.69
-----------------
-----------------
Card ID: 71941543
Balance: $149.34
-----------------
-----------------
Card ID: 77021664
Balance: $53.16
-----------------
-----------------
Card ID: 80390967
Balance: $76.42
-----------------
-----------------
Card ID: 81451080
Balance: $59.13
-----------------
-----------------
Card ID: 81839215
Balance: $77.84
-----------------
-----------------
Card ID: 83628056
Balance: $210.04
-----------------
-----------------
Card ID: 88057904
Balance: $210.03
-----------------
-----------------
Card ID: 91557921
Balance: $224.68
-----------------
-----------------
Card ID: 92502419
Balance: $120.37
-----------------
-----------------
Card ID: 94667151
Balance: $138.05
-----------------
-----------------
Card ID: 95629628
Balance: $103.27
-----------------
Enter command (a,g,p,q,r, h for Help)> Number of cards on file: 41
Average balance: $131.50
Enter command (a,g,p,q,r, h for Help)> Bye.
